### PR TITLE
changes after deploy prod-setting page

### PR DESCRIPTION
**Pull request by raven41k**

### CODE REVIEW

File: client/src/pages/SettingsPage/Sources/Sources.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ const Sources = ({ sources, updateSources, toggleSources }) => {
                   value={sources[name]}
                   onChange={updateSources}
                 />
-                <a target="_blank" href={text}>{text}</a>
+                <a target="_blank" href={`//${text}`}>{text}</a>
```
Comment (happylolonly) [inline client/src/pages/SettingsPage/Sources/Sources.js]: Rename 'text' to 'link' please.

Comment (happylolonly) [inline client/src/pages/SettingsPage/Sources/Sources.js]: Why '//' better than 'www.' ?

**Review by happylolonly — CHANGES_REQUESTED** [review]

File: client/src/pages/SettingsPage/Sources/Sources.js (hunk 1 above)
Comment (raven41k) [reply client/src/pages/SettingsPage/Sources/Sources.js]: '//' works correctly

**Comment by happylolonly** [comment]
fix #25

**Review by happylolonly — APPROVED** [review]

**Pull request merged**